dev-lang/python/python-3.10.3.ebuild

```diff
@@ -236,6 +236,11 @@ src_compile() {
 	# https://bugs.gentoo.org/823728
 	export SETUPTOOLS_USE_DISTUTILS=stdlib
 
+	# Save PYTHONDONTWRITEBYTECODE so that 'has_version' doesn't
+	# end up writing bytecode & violating sandbox.
+	# bug #831897
+	local -x _PYTHONDONTWRITEBYTECODE=${PYTHONDONTWRITEBYTECODE}
```

Comment:
Is there any reason to actually export it and not just keep it as a short local var?

Reply:
No, we can keep it as a short local. Any preference for name?

Reply:
`orig_bytecode`?